workflow/snakemake_rules/main_workflow.smk

```diff
@@ -1462,28 +1446,10 @@ rule export:
             --output {output.auspice_json} 2>&1 | tee {log}
         """
 
-rule add_branch_labels:
-    message: "Adding custom branch labels to the Auspice JSON"
-    input:
-        auspice_json = rules.export.output.auspice_json,
-        emerging_clades = rules.emerging_lineages.output.clade_data
-    output:
-        auspice_json = "results/{build_name}/ncov_with_branch_labels.json"
-    log:
-        "logs/add_branch_labels{build_name}.txt"
-    conda: config["conda_environment"]
-    shell:
-        """
-        python3 ./scripts/add_branch_labels.py \
```

Comment:
Nice to see these temp rules deleted! Can we also delete `add_branch_labels.py` now that we don't use it anywhere else? We could always revive it later from Git, if we needed it...

Reply:
[Done](https://github.com/nextstrain/ncov/compare/21f88a250da3e30191fee54ac946b12128b86dd4..322b8a9076b93c676b062909fa37f1905a665161)

For reference (from the updated commit message):
> Note that [the script] contained unused functionality to export spike mutations; if we reinstate this in the future we should update the output format to produce a node-data JSON with a custom branch label to avoid modifying the auspice JSON after export.